modules/gapi/src/backends/ov/govbackend.cpp

```diff
@@ -252,7 +252,8 @@ class OVCallContext
                   const std::vector<cv::gimpl::RcDesc>              &  outs,
                   cv::GRunArg::Meta                                 && meta,
                   std::vector<cv::gimpl::GIslandExecutable::InObj>  && input_objs,
-                  std::vector<cv::gimpl::GIslandExecutable::OutObj> && output_objs);
+                  std::vector<cv::gimpl::GIslandExecutable::OutObj> && output_objs,
+                  const bool                                           inference_only);
```

Comment:
probably you can omit it from there - as it is not the mandatory parameter.

it can be `false` internally by default.

Reply:
This is the place where `inference_only` from `GOVExecutable` is propogated to the specific inference `kernel` (e.g Infer, InferList, etc):
https://github.com/TolyaTalamanov/opencv/blob/at/implement-inference-only-mode-for-ov-backend/modules/gapi/src/backends/ov/govbackend.cpp#L1501-L1502

I guess there is no sense to do something like that:
```
auto ctx = std::make_shared<OVCallContext>(uu, out, op.args, op.outs,
            std::move(stub_meta), std::move(input_objs), std::move(output_objs));
if (m_inference_only) {
    ctx->enableInferenceOnly();
}
```
It's easier to pass this straight to context ctor.

Reply:
It should remain in the class but it shouldn't be a mandatory constructor argument